engine/storage_mongo_datadb.go

```diff
@@ -1745,10 +1745,7 @@ func (ms *MongoStorage) SetDispatcherProfileDrv(r *DispatcherProfile) error {
 
 func (ms *MongoStorage) RemoveDispatcherProfileDrv(tenant, id string) error {
 	return ms.query(func(sctx mongo.SessionContext) error {
-		dr, err := ms.getCol(ColDpp).DeleteOne(sctx, bson.M{"tenant": tenant, "id": id})
-		if dr.DeletedCount == 0 {
```

Comment:
Why was removal here necessary, did we get any kind of issues with it?

Thanks!